### PR TITLE
chore: update to webpack 2 and delete useless now files

### PR DESCRIPTION
**Pull request by Alwox**

### CODE REVIEW

File: cfg/dist.js [hunk 1]
```diff
@@ -1,42 +0,0 @@
-'use strict';
```
Comment (pjazdzewski1990) [inline cfg/dist.js:1]: How will you split execution between `local` (unoptimized, for development) and `dist`(production version) ?

**Review by pjazdzewski1990 — APPROVED** [review]

File: cfg/dist.js (hunk 1 above)
Comment (Alwox) [reply cfg/dist.js:1]: This divide is not implemented yet, I have only set generate bundle.js in `dist` folder and copy there rest of files. My first goal was to create a working example, now I'm reading how to develop this.

Comment (piecyk) [reply cfg/dist.js:1]: @pjazdzewski1990 it should be provided by webpak 

basic we pass as node env to webpack config  
```js
//  build consts 
const buildTimestamp = Date.now();
const nodeEnv = process.env.NODE_ENV;
const isDevelopmentMode = nodeEnv !== 'production';
const uiBaseUrl = process.env.UI_BASE_URL || '';
```
then later in plugins we push 
```js
new webpack.DefinePlugin({
  // values are treated as code fragments so add quotes inside the value
  'process.env': _.mapValues({
    IS_DEVELOPMENT_MODE: isDevelopmentMode,
    UI_BASE_URL: uiBaseUrl,
    BUILD_TIMESTAMP: buildTimestamp,
    NODE_ENV: nodeEnv // used by React for example
    // etc... 
  }, v => JSON.stringify(v))
}),
```
and in runtime you can asses it by process.env, 
like `process.env.IS_DEVELOPMENT_MODE;`

**Comment by piecyk** [comment]
yeah 👏 finally the yo crap is gone 👍

**Comment by Alwox** [comment]
I'm also happy with that - I don't like 'magic parts of code' which I don't understand 😉

File: cfg/dist.js (hunk 1 above)
Comment (pjazdzewski1990) [reply cfg/dist.js:1]: Ok, that answers my question

**Pull request merged**